test/generate-json.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import time
+import os
+import struct
+
+# PingMessage is generated by this project
+import sys
+sys.path.append("../src/protocol/python")
+import PingMessage as Message
+
+# argparse for future devices
+parser = argparse.ArgumentParser(description="Script to generate json test description file for ping protocol!")
+parser.add_argument('--device', action="store", type=str, default="Ping1D", help="Ping device type.")
+args = parser.parse_args()
+
+# Generate fake data to decode
+def _generateFakeData(format):
+    return bytearray(os.urandom(struct.calcsize(format)))
+
+# Create payload
+def _values(format, fakedata, payload_fields):
+    values = list(struct.unpack(format, fakedata))
+    data = {}
+    for a,b in zip(values, payload_fields):
+        data[b] = a
+    return data
+
+#Checksum = sum(0 -> n) & 0xffff
```

Comment:
Why not docstrings instead of comments? even the one-line one should be good.
See [PEP257](https://www.python.org/dev/peps/pep-0257/)